src/mono/wasm/tools/create-pack.proj

```diff
@@ -0,0 +1,74 @@
+<Project>
```

Comment:
@radical is your intent to do away with some of https://github.com/dotnet/runtime/tree/master/src/mono/nuget ?

Reply:
This is just for quick local setup for testing.